src/rill_query.c

```diff
@@ -84,7 +84,11 @@ int main(int argc, char *argv[])
         if (!store) rill_exit(1);
 
         if (key) pairs = rill_store_query_key(store, key, pairs);
-        else pairs = rill_store_scan_vals(store, &val, 1, pairs);
+
+        // TODO this should be implemented with new query_vals
+        // query_vals vs scan vals meaning
```

Comment:
FYI a raptor tackled me before I was able to finish this sentence. I think where I was going at wa